sssom/cli.py

```diff
@@ -31,51 +52,86 @@ def main(verbose, quiet):
 
 
 @main.command()
-@click.option('-i', '--input')
-@click.option('-f', '--format')
-@click.option('-o', '--output')
-@click.option('-t', '--to-format')
-@click.option('-c', '--context')
+@click.option('-i', '--input', help=help_input)
+@click.option('-f', '--format', help=help_input_format)
+@click.option('-o', '--output', help= help_output)
+@click.option('-t', '--to-format', help=help_format)
+@click.option('-c', '--context', help=help_context)
 def convert(input: str, output: str, format: str, to_format: str, context: str):
-    """
-    convert file (currently only supports conversion to RDF)
-    """
+    """Convert file (currently only supports conversion to RDF)
+
+    Args:
```

Comment:
use consider `\b` for formatting

Reply:
also, we should use one docstring scheme throughout; this looks like the google one, which is different from the one used elsewhere.. I don't care which, just make sure its the same throughout.

Reply:
I agree. But since we will be using this space for examples as Chris suggested before, this would look like:

```
[Description]

[Example]
```

Does the doctoring scheme differentiate how it displays?

Reply:
I dont know - I wouldnt think so, but probably worth a google (especially for systems that use docstrings that autogenerate cool yaml docs).

Reply:
Tried `\b`, does not work:
<img width="609" alt="Screen Shot 2021-05-27 at 10 56 15 AM" src="https://user-images.githubusercontent.com/5247075/119858719-57caf980-beda-11eb-8942-14831715a489.png">

I am replacing everything under `Args` (including Args) by `Examples`. I will need help from @cmungall and/or @matentzn to fill in the blanks [A B C ... as placeholders below]

<img width="527" alt="Screen Shot 2021-05-27 at 10 59 48 AM" src="https://user-images.githubusercontent.com/5247075/119859087-ac6e7480-beda-11eb-9d03-e160af200a26.png">

Reply:
We will do this bit by bit in a different PR